Reference/Templating/Mvc/views.md

```diff
@@ -10,82 +10,56 @@ _Working with MVC Views and Razor syntax in Umbraco_
 
 ```csharp
-@Umbraco.Field(Model.Content, "eventLink")
+@inherits UmbracoViewPage<SiteSettings>
+...
+@Model.Value("title")
 ```
 
 You will also need to pass the "Context" to the @Umbraco.Field() method if you're looping over a selection like this where we pass the "item" variable.
 
+Lopping over a selection works in a similar way. If you have a property that contains, for instance, an IEnumberable collection, you can access the individual items using a foreach loop. Below illustrates how you might do that using "item" as a variable.
+
 ```csharp
 @{
-    var selection = Model.Content.Site().FirstChild("events").Children("event").Where(x => x.IsVisible());
+    var collection = Model.ItemList;
 }
 
 <ul>
-    @foreach(var item in selection){
-        @Umbraco.Field(item, "eventLink")
+    @foreach(var item in collection){
+        <p>@item.Name</p>
     }
 </ul>
 ```
-
-There are several optional parameters. Here is the list with their default values:
-
-* `altFieldAlias = ""`
-* `altText = ""`
-* `insertBefore = ""`
-* `insertAfter = ""`
-* `recursive = false`
-* `convertLineBreaks = false`
-* `removeParagraphTags = false`
-* `casing = RenderFieldCaseType.Unchanged`
-* `encoding = RenderFieldEncodingType.Unchanged`
-
-The easiest way to use the Field method is to specify the optional parameters you'd like to set. For example, if we want to set the insertBefore and insertAfter parameters we'd do:
-
-```csharp
-@Umbraco.Field("bodyContent", insertBefore : "<h2>", insertAfter : "</h2>")
-```
-
-## Rendering a field with Model
-
-The UmbracoHelper method provides many useful parameters to change how the value is rendered. If you however want to render value "as-is" you can use the @Model.Content property of the view. For example:
-
-```csharp
-@Model.Content.Properties["bodyContent"].Value
-```
-
-Or alternatively:
-
-```csharp
-@Model.Content.GetPropertyValue("bodyContent")
-```
-
-You can also specify the output type that you want from the property. If the property editor or value does not support the conversion then an exception will be thrown. Some examples:
-
-```csharp
-@Model.Content.GetPropertyValue<double>("amount")
-@Model.Content.GetPropertyValue<RawXElement>("xmlContents")
-```
+If you want to convert a type and it's possible, you can do that by typing a variable and assigning the value from your property to it. This might could look like the example below.
+`
+@foreach (TeamMember person in Model.TeamMembers)
+                        {
+                           <a href="@person.Url">                                                              
+                             <p>@person.Name</p>
+                           </a>
+                         }
+ `
+
+In this example, we are looping through a list of items with the custom made type TeamMember assigned. This means we are able to access the strongly typed properties on the TeamMember item.
 
 ## Rendering a field using @CurrentPage (dynamically)
```

Comment:
I would update the warning to mention that dynamic has been removed (and remove the info on how to use it)- but leave this info in a V7 version of this documentation page

Reply:
Versioning... I want to have a v7 version and a v8. I am kinda writing a v8 one. Should I change the version in the meta data to 8 and then create a new file for 7? Or should I keep this as 7 and then create a new v8?

Reply:
create a views-v7.md file and have this containing V7 specific info with the YAML saying From 7.0.0 then in then in the file you are editing view.md file it can be totally V8 focussed with the YAML saying from 8.0.0   ... that way this info is presented first without the noise of legacy information, but the dropdown allows people to see the V7 version... if they wish... it is a little complicated to understand and hastily cobbled together... but so far hasn't completely broken down!